packages/graphql/schema.graphql

```diff
@@ -103,7 +103,7 @@ type Mutation {
   launchOpenProject: App
 
   """load saved projects into context"""
-  loadProjects: App
+  loadProjects: App!
```

Comment:
I'm not sure if I'd expect "loadProjects" to be a mutation, as much as just something that happens implicitly as part of starting the application. Wanted to know if there was a different expectation around this I was missing.

Reply:
I originally had this data initialize in the server context instantiation, but there was some concern about that (see Lachlan's comment above). I could go either way on this. We could initially populate the data for this at the start of the application or we could populate when the global mode page mounts. I might not have the whole picture, but it seems to me that the projects are only important when you're in global mode.

Reply:
I will add this to our graphql sync meeting as a point of discussion. Basically, the underlying problem I'm struggling with is "how to tell graphql something is loading that isn't initially triggered from graphql". There must be a pattern.

Reply:
Oh - I think this should be a query, not a mutation (as opposed to something we kick off inside of `constructor`)

Reply:
Yah that makes a lot of sense that this should be a query. At the end of the day the consumer shouldn't care about how to load the data and it's more of an implementation detail that we need to load the data. When they say give me the projects the resolver should figure out if we need to hit the cache or not.